### PR TITLE
Prettier and Eslint configuration

### LINKED ISSUE #10: Set up prettier and eslint

### PR DESCRIPTION
**Pull request by natalieacevedo**
-Set up of ESLint and Prettier, I disabled ESLlint when overlapping.
-Created file for ignoring commits related to code formatting.

Fixes #10

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+PUBLIC_URL=http://localhost:3000/
```
Comment (haplo) [inline .env]: This should go in `.env.sample`, please add `.env` to `.gitignore` and take it out of the commits.

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": false,
+  "printWidth": 80
```
Comment (haplo) [inline .prettierrc.json]: Let's make it 90.

**Review by haplo — CHANGES_REQUESTED** [review]
Please remove _package-lock.json_, we use _yarn_ so _yarn.lock_ is enough.

**Comment by natalieacevedo** [comment]
> Please remove _package-lock.json_, we use _yarn_ so _yarn.lock_ is enough.



> Please remove _package-lock.json_, we use _yarn_ so _yarn.lock_ is enough.
It's already removed

File: .env (hunk 1 above)
Comment (natalieacevedo) [reply .env]: done!

File: .prettierrc.json (hunk 2 above)
Comment (natalieacevedo) [reply .prettierrc.json]: ok!

File: .env [hunk 3]
```diff
@@ -1,3 +1,3 @@
-PUBLIC_URL=http://localhost:3000/
-
+REACT_APP_PUBLIC_URL=http://localhost:3000/
```
Comment (haplo) [inline .env]: Please remove `.env` from git.

File: .gitignore [hunk 4]
```diff
@@ -12,8 +12,11 @@
 /build
 
 # misc
+.env
 .DS_Store
 .env.local
+
+
```
Comment (haplo) [inline .gitignore]: Remove these blank lines.

File: src/pages/AlbumDetailPage.jsx [hunk 5]
```diff
@@ -6,21 +6,22 @@ import AlbumList from "../components/AlbumList";
 import PhotoList from "../components/PhotoList";
 
 function fetchAlbum(albumId) {
+  console.log(process.env);
```
Comment (haplo) [inline src/pages/AlbumDetailPage.jsx]: Remove `console.log`

**Review by haplo — CHANGES_REQUESTED** [review]
Please remove `package-lock.json`

**Review by haplo — APPROVED** [review]

**Pull request merged**